tapestry.js

```diff
@@ -4,13 +4,15 @@
  * CONSTANTS AND GLOBAL VARIABLES
  ****************************************************/
 
+
+
 const // declared
     TAPESTRY_CONTAINER_ID = "tapestry",
-    PROGRESS_THICKNESS = 20,
+    PROGRESS_THICKNESS = (20/1.5),  // RM /100 - ford
     LINK_THICKNESS = 6,
-    NORMAL_RADIUS = 140,
-    ROOT_RADIUS_DIFF = 70,
-    GRANDCHILD_RADIUS_DIFF = -100,
+    NORMAL_RADIUS = (140/1.5),  //140 originally
+    ROOT_RADIUS_DIFF = (70/1.5), //70 originally
+    GRANDCHILD_RADIUS_DIFF = (-100/1.5), //-100 originally
```

Comment:
Might make sense to change all of these into variables (instead of constants) and then create another variable for `1.5` that you then multiply by those other variables. That way, it can be easily adjustable.